setup.py

```diff
@@ -0,0 +1,42 @@
+"""Setup for ai_tutor XBlock."""
+
+
+import os
+
+from setuptools import setup
+
+
+def package_data(pkg, roots):
+    """Generic function to find package_data.
+
+    All of the files under each of the `roots` will be declared as package
+    data for package `pkg`.
+
+    """
+    data = []
+    for root in roots:
+        for dirname, _, files in os.walk(os.path.join(pkg, root)):
+            for fname in files:
+                data.append(os.path.relpath(os.path.join(dirname, fname), pkg))
+
+    return {pkg: data}
+
+
+setup(
+    name='ai_tutor-xblock',
```

Comment:
We should be consistent with name of xblock everywhere in the code and files names. what if we use `ai_coach` everywhere?